solutions/silver/cses-1619.mdx

```diff
@@ -301,4 +358,48 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+<Warning>
+
+Due to Python's big constant factor, this code may TLE.
```

Comment:
wait does it only tle sometimes?

Reply:
No, it will tle all the time if you run it on cses. I think some people run python/java code on codeforces to get more forgiving verdicts, so I put may TLE.